wire-ios-system/Source/Logging/CocoaLumberjackLogger.swift

```diff
@@ -73,6 +73,8 @@ final class CocoaLumberjackLogger: LoggerProtocol {
         //    return
         // }
 
+        guard isDebug || (level.rawValue <= DDLogLevel.info.rawValue) else { return }
```

Comment:
so we log isDebug or everything under info, don't you mean >= info?

Reply:
The raw values of `DDLogLevel` are as follows:
```
DDLogLevelOff     00000 0
DDLogLevelError   00001 1
DDLogLevelWarning 00011 3
DDLogLevelInfo    00111 7
DDLogLevelDebug   01111 15
DDLogLevelVerbose 11111 31
DDLogLevelAll  1..11111 UInt.max
```

Reply:
ah makes sense, maybe add a comment then, it's sure not obvious to me